src/styles/main.scss

```diff
@@ -1,3 +1,21 @@
-body {
+@import "./utils/vars";
+@import "./utils/mixins";
+@import "./blocks/header";
+@import "./blocks/card";
+@import "./blocks/stars";
+
+* {
   margin: 0;
+  transition-duration: 0.3s;
```

Comment:
it is better to apply transition-duration only to elements whose properties change